storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.25-17.0.rst

```diff
@@ -0,0 +1,44 @@
+.. _8.0.25-17:
+
+
+================================================================================
+*Percona XtraBackup* 8.0.25-17.0
+================================================================================
+
+:Date: June 2, 2021
+:Installation: `Installing Percona XtraBackup <https://www.percona.com/doc/percona-xtrabackup/8.0/installation.html>`_
+
+Percona XtraBackup enables MySQL backups without blocking user queries, making it ideal
+for companies with large data sets and mission-critical applications that cannot tolerate
+long periods of downtime. Offered free as an open source solution, it drives down backup
+costs while providing unique features for MySQL backups.
+
+Bugs Fixed
+================================================================================
+
+* :jirabug:`PXB-2442` : Backup cannot be decompressed using the apparmor profile
+
+* :jirabug:`PXB-2444` : Xbcloud fails to upload backup with enforcing SELinux mode
+
+* :jirabug:`PXB-2505` : Keyring is not generated properly by generate-new-master-key for component
```

Comment:
Remove this one, this is a post fix after the merge

Reply:
removed.